.gitignore

```diff
@@ -26,3 +26,14 @@
 
 coverage
 tmp
+
+# ignore generated assets
+public/assets/component_guide
+public/assets/govuk_publishing_components
+public/assets/*.css
```

Comment:
It seems like we do have [one non-autogenerated asset in here](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/blob/master/public/assets/error-pages.85f433bf6332a1245653b92b44d2430a55810be0.min.css) 😬

Reply:
😬 I'll fix that.

Reply:
That was quick. Thank you!